apps/sharedtest/test/unit/sticky_header_test.js

```diff
@@ -129,14 +129,14 @@ suite('shared/js/sticky_header.js', function() {
       checkBackgroundImage('header_0');
     });
 
-    test('Scroll 1280: background set to the second header', function() {
-      scrollAndCheckBackgroundImage.bind(this)(1280, 'header_2');
+    test('Scroll 1280: background set to the second header', function(done) {
+      scrollAndCheckBackgroundImage.bind(this)(1280, 'header_2', done);
     });
 
-    test('Scroll 1280: background set to the first header', function() {
+    test('Scroll 1280: background set to the first header', function(done) {
       var header2 = document.getElementById('header_2');
       header2.hidden = true;
-      scrollAndCheckBackgroundImage.bind(this)(1280, 'header_1');
+      scrollAndCheckBackgroundImage.bind(this)(1280, 'header_1', done);
       delete header2.hidden;
```

Comment:
Seems strange to have this still after calling `done`. I don't think you added it though, so probably fine to leave it if you don't want to change it.